lib/routes/bitget/announcement.ts

```diff
@@ -0,0 +1,194 @@
+import { DataItem, Route } from '@/types';
+import ofetch from '@/utils/ofetch';
+import { load } from 'cheerio';
+import cache from '@/utils/cache';
+import { BitgetResponse } from './type';
+import { parseDate } from '@/utils/parse-date';
+
+const handler: Route['handler'] = async (ctx) => {
+    const baseUrl = 'https://www.bitget.com';
+    const announcementApiUrl = `${baseUrl}/v1/msg/push/stationLetterNew`;
+    const { type, lang = 'zh-CN' } = ctx.req.param<'/bitget/announcement/:type/:lang?'>();
+    const languageCode = lang.replace('-', '_');
+    const headers = {
+        Referer: baseUrl,
+        accept: 'application/json, text/plain, */*',
+        'content-type': 'application/json;charset=UTF-8',
+        language: languageCode,
+        locale: languageCode,
+    };
+    const pageSize = ctx.req.query('limit') ?? '10';
+
+    // stationLetterType: 0 表示全部通知，02 表示新币上线，01 表示最新活动，06 表示最新公告
+    const reqBody: {
+        pageSize: string;
+        openUnread: number;
+        stationLetterType: string;
+        isPre: boolean;
+        lastEndId: null;
+        languageType: number;
+        excludeStationLetterType?: string;
+    } = {
+        pageSize,
+        openUnread: 0,
+        stationLetterType: '0',
+        isPre: false,
+        lastEndId: null,
+        languageType: 1,
+    };
+
+    // 根据 type 判断 reqBody 的 stationLetterType 的值
+    switch (type) {
+        case 'new-listing':
+            reqBody.stationLetterType = '02';
+            break;
+
+        case 'latest-activities':
+            reqBody.stationLetterType = '01';
+            break;
+
+        case 'new-announcement':
+            reqBody.stationLetterType = '06';
+            break;
+
+        case 'all':
+            reqBody.stationLetterType = '0';
+            reqBody.excludeStationLetterType = '00';
+            break;
+
+        default:
+            throw new Error('Invalid type');
+    }
+
+    const response = (await cache.tryGet(`bitget:announcement:${type}:${pageSize}:${lang}`, async () => {
+        const result = await ofetch<BitgetResponse>(announcementApiUrl, {
+            method: 'POST',
+            body: reqBody,
+            headers,
+        });
+        if (result?.code !== '200') {
+            throw new Error('Failed to fetch announcements, error code: ' + result?.code);
+        }
+        return result;
+    })) as BitgetResponse;
```

Comment:
Caching this request is not suggested. If you really want to cache it, please set the 3rd parameter of [`tryGet`](https://docs.rsshub.app/joinus/advanced/use-cache#api) to `config.cache.routeExpire` and 4th to `false`

Reply:
> Caching this request is not suggested. If you really want to cache it, please set the 3rd parameter of [`tryGet`](https://docs.rsshub.app/joinus/advanced/use-cache#api) to `config.cache.routeExpire` and 4th to `false`

Because if this endpoint is accessed too frequently, a 403 error will occur. I'm conflicted about whether to use it. Do you have any suggestions?

Reply:
> If you really want to cache it, please set the 3rd parameter of [`tryGet`](https://docs.rsshub.app/joinus/advanced/use-cache#api) to `config.cache.routeExpire` and 4th to `false`

Reply:
> > If you really want to cache it, please set the 3rd parameter of [`tryGet`](https://docs.rsshub.app/joinus/advanced/use-cache#api) to `config.cache.routeExpire` and 4th to `false`

I have changed it :)